app/javascript/components/admin/AdminTable.js

```diff
@@ -57,6 +59,33 @@ class AdminTable extends React.Component {
     this.csvLink = React.createRef();
   }
 
+  handleExpandClick = rowId => {
+    let ids = this.state.expandedNotesIds;
+    ids.indexOf(rowId) > -1 ? ids.splice(ids.indexOf(rowId), 1) : ids.push(rowId);
+    this.setState({ expandedNotesIds: ids });
+  };
+
+  /**
+   * Creates a "Collapse/Expand" button for each row of the table.
+   * @param {Object} rowData - Data about the cell this filter is called on.
+   */
+  renderNotes = data => {
+    const rowData = data.rowData;
+    return (
+      <div id={rowData.id}>
+        {rowData.notes.length < 200 && <div>{rowData.notes}</div>}
```

Comment:
might be worth creating a reusable component for this as it's used in some other places as well? @sgober

Reply:
It might be, I feel like there would be quite a few props involved though.  I am okay either way

Reply:
resolving this because this change isn't absolutely necessary